shm/Game.cpp

```diff
@@ -14,10 +14,10 @@ bool Game::checkLoseCondition() const {
 
 void Game::PrintOptions() {
     std::cout << "Ahoy captain! We're waiting for your commands! \n";
-    std::cout << "0. Exit \n";
     std::cout << "1. Travel \n";
     std::cout << "2. Buy \n";
     std::cout << "3. Sell \n";
+    std::cout << "0. Exit \n";
```

Comment:
Let's just go with 4 instead of 0.

Reply:
I think that 0 for exit it's a good idea. If we want to implement another options to menu, like shipyard to sell/buy ships, each time we will move exit to next number. It isn't a big deal to change number for exit, but 0 will be much constantly.

Reply:
I have the same way of thinking like Artur:)